spec/helpers/project_controller_helper_spec.rb

```diff
@@ -31,12 +31,18 @@
     end
 
     before do
-      project.stub(:causes) { [double('Tag', name: 'Cause1'), double('Tag', name: 'Cause2')] }
-      project.stub(:technologies) { [double('Tag', name: 'Ruby'), double('Tag', name: 'Rails')] }
+      project.stub(:causes) do
+        [double('Tag', name: 'Cause1'), double('Tag', name: 'Cause2')]
+      end
+      project.stub(:technologies) do
+        [double('Tag', name: 'Ruby'), double('Tag', name: 'Rails')]
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.